incremental-font-transfer/src/bin/ift_graph.rs

```diff
@@ -199,15 +203,24 @@ fn to_next_font(
         .expect("Patch application failed."))
 }
 
+#[derive(Clone, Default, Ord, PartialEq, PartialOrd, Eq)]
+struct NodeName(String);
```

Comment:
Are these frequently cloned? If so a type other than `String` might be preferable.